tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectUtil.java

```diff
@@ -1209,6 +1209,18 @@ public static boolean isExtProject( IProject project )
         return hasFacet( project, IPluginFacetConstants.LIFERAY_EXT_PROJECT_FACET );
     }
 
+    public static boolean isGradleProject(String location)
+    {
+        boolean retval = false;
+        File build = new File( location, "build.gradle" );
```

Comment:
Not always... there are some gradle projects that don't have "build.gradle" in them.  Also this code should likely be in a gradle.core plugin and not in project.core